cocos/ui/UILayout.h

```diff
@@ -401,7 +401,7 @@ class CC_GUI_DLL Layout : public Widget, public LayoutProtocol
     /**
      * request to refresh widget layout
      */
-    void requestDoLayout();
+    virtual void requestDoLayout();
```

Comment:
You mistook it. It is not overriding. `requestDoLayout()` is defined here in `Layout` at first.